src/tests/end-to-end/tests/target-page/visualization-boxes.test.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { Browser } from '../../common/browser';
+import { launchBrowser } from '../../common/browser-factory';
+import { PopupPage } from '../../common/page-controllers/popup-page';
+import { TargetPage } from '../../common/page-controllers/target-page';
+import { scanForAccessibilityIssues } from '../../common/scan-for-accessibility-issues';
+
+describe('Target Page visualization boxes', () => {
+    let browser: Browser;
+    let targetPage: TargetPage;
+    let popupPage: PopupPage;
+
+    beforeAll(async () => {
+        browser = await launchBrowser({ suppressFirstTimeDialog: true });
+        targetPage = await browser.newTargetPage();
+        popupPage = await browser.newPopupPage(targetPage);
+        await popupPage.gotoAdhocPanel();
+    });
+
+    beforeEach(async () => {
+        await popupPage.disableAllToggles();
```

Comment:
I'm not sure this is guaranteed to wait for the old visualizations to actually disappear from the page. Consider adding a `TargetPage.waitForVisualizationsToDisappear()` or something (it'll probably want to accept either of "the root container is not present" or "the shadow root is present but empty")